neo/Ledger/Blockchain.cs

```diff
@@ -181,6 +186,13 @@ public Blockchain(NeoSystem system, Store store)
             }
         }
 
+        internal async Task<Done> ShutdownAndWaitForIdle()
```

Comment:
We can just throw away the remaining messages when shutting down.

Reply:
I'll implement something that does this to speed up the shutdown process then.

Reply:
So, rather than explicitly throwing away the remaining, message, I decided to go back to relying on using `Stop` to stop the actors, since that won't process the remaining messages. Instead of waiting until the `Blockchain` is Idle, I just wait for the actor to actually be stopped. This should ensure the order of actors being destroyed doesn't cause Exceptions while shutting down the system.